buzzmobile/process/gps_mapper/gps_mapper.py

```diff
@@ -36,11 +36,14 @@
 X_SCALE = Y_SCALE = 1000 * rospy.get_param('pixels_per_m')
 LINE_WIDTH = int(round(rospy.get_param('pixels_per_m')
                        * rospy.get_param('road_width')))
+IMAGE_WIDTH = rospy.get_param('image_width')
+IMAGE_HEIGHT = rospy.get_param('image_height')
+IMINFO = Image(LINE_WIDTH, IMAGE_HEIGHT, IMAGE_WIDTH)
```

Comment:
Also shouldn't this line say `interpolate.Image`, since we're importing interpolate, and not `from interpolate import *`?